src/sidebar/test/virtual-thread-list-test.js

```diff
@@ -36,8 +36,17 @@ describe('VirtualThreadList', function() {
   }
 
   beforeEach(() => {
+    const fakeDebounce = callback => {
+      const debounced = () => {
+        // Update synchronously instead of really debouncing.
+        callback();
+      };
+      debounced.cancel = sinon.stub();
+      return debounced;
+    };
```

Comment:
This change is just about adding a `cancel` method to the result of the `debounce` mock.